sopel/config/core_section.py

```diff
@@ -545,6 +596,28 @@ class CoreSection(StaticSection):
     .. versionadded:: 7.0
     """
 
+    flood_text_length = ValidatedAttribute('flood_text_length', int, default=50)
+    """Length of text at which an extra wait penalty is added.
+
+    :default: ``50``
+
+    Message that are longer than this length get an added wait penalty if the
+    food protection limit is reach.
```

Comment:
```suggestion
    Messages longer than this (in bytes) get an added wait penalty if the
    flood protection limit is reached.
```

Some days I feel like I need "food protection" to keep me from overeating. 😂